### DISCUSSION TITLE
test system format

### DISCUSSION THREAD

**Discussion by LeZhengThu** [CURENT/andes#147] [Data Format] [answered]
Dr. Cui, 
       I notice that the test system data are contained in *.raw and *.dyr files. It seems that InterPSS also uses those two extensions to define the test system. I wonder what's the original software that defined these two extensions. Where can I find the documentation of the two extensions? I'd like to convert my own system in order to use ANDES or InterPSS. Thanks.

**Comment by cuihantao** [comment, marked as answer]
`raw` and `dyr` are introduced by PTI PSS/E. You can find more details from PSS/E's documentations.

**Comment by LeZhengThu** [comment]
@cuihantao Thank you, Dr. Cui.

**Comment by LeZhengThu** [comment]
@cuihantao Dr. Cui, I don't have a valid PSS/E license. And I can't find the data definition and specification for the *.dyr file anywhere online. Could you please provide some documentations for the *.dyr file? Thanks.

**Comment by cuihantao** [comment]
Hi @LeZhengThu ,

The document is proprietary to Siemens PTI, so I could not share the document with you.

One way is to find trial versions of PSS/E (known as PSS/E explore). After installing the software, you can find the documents in the `DOCS` folder. Specifically, the dynamic model definitions are in `MODELS.pdf`.

You can try requesting PSSE Explore at https://new.siemens.com/global/en/products/energy/energy-automation-and-smart-grid/pss-software/psse-xplore-order-form.html

**Comment by cuihantao** [comment]
Since this is not an issue, I'm converting it into a discussion. You can find the discussion thread from "Discussions" on the right of the "Pull Request" tab.
